k_p_xy:   192.0
k_i_xy:   10.0
k_d_xy:   156.0
k_a_xy:   15.0

k_p_z:    99.0
k_i_z:    0.0
k_d_z:    46.0
k_a_z:    10.0

k_p_yaw:  1.0
k_i_yaw:  1.0
k_d_yaw:  1.0
k_a_yaw:  1.0
